packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -166,16 +166,16 @@ export class ColorPickerBase extends BaseComponent<IColorPickerProps, IColorPick
   };
 
   private _onHexChanged = (): void => {
-    this._updateColor(getColorFromString('#' + this.hexText.value));
+    this._updateColor(getColorFromString('#' + this._hexText.value));
```

Comment:
I believe `value` is deprecated. Use `current`.

(We REALLY need to enable the `deprecations` tslint rule.

Reply:
This is all pre-existing code. The only reason this changed at all is because TextField is now in an HOC wrapper. I'd rather not make ref-related or deprecation changes to this PR as it's already pretty wide in scope.